models/tridimensional/docking_validation/batch_dock.sh

```diff
@@ -55,7 +115,7 @@ while [ $i -lt $threads ]; do
 	err_name=$log_name
 	err_name+="_err.txt"
 
-	python $SCRIPTS/dock_variants.py -s=$pam_first -e=$pam_last --output_dir=$results_dir --pdb_dir=$pdb_dir > $out_name 2> $err_name &
+	nohup nice -n 10 python $SCRIPTS/dock_variants.py -s=$pam_first -e=$pam_last --output_dir=$results_dir --pdb_dir=$pdb_dir $opt_dock_variants_args > $out_name 2> $err_name &
```

Comment:
we may want a nohup and a nice input argument too, in case we run this locally / on our own server (lol)